### ISSUE TITLE
[enhancement] Update rover flags to be more consistent with Terraform

### ISSUE THREAD

**Issue by Justin-DynamicD** [aztfmod/rover#81]
RIght now Rover is essentially a helper/wrapper for terraform that manages things to work better with the launchpad.  However, rover uses a completely unique syntax in order to interact with it.

I think it would speed adoption if rover use was in parity with terraform use, at least the key features.

Example: don`t require `-lz`  or `-a` in order to select a landingzone and action (especially as its just essentially just a project). Allow [dir] to be specified if desired, or just run off the current directory if not (you can read full path and get the "landingzone" name anyway).

These little differences trip things up a lot. Look at the following, especially if Ive already browsed into the project folder in question:

```
terraform plan

# vs

rover -a plan -lz /tf/caf/full/path/

# but image this 

rover plan
```

This would make rover a bit more "terragrunt-like", but I dont thinkj thats a bad thing.

**Comment by arnaudlh** [comment]
Absolutely @Justin-DynamicD, good idea, next major update of rover will implement that.

**Comment by nusrath432** [comment]
@Justin-DynamicD Good idea @arnaudlh  would this be implemented in the next major rover release or next major rovergo?
